tests/workbenches/notebook-controller/conftest.py

```diff
@@ -0,0 +1,282 @@
+    kueue_enabled_notebook_namespace: Namespace,
+) -> Generator[LocalQueue, Any, Any]:
+    """Create a LocalQueue for notebook workloads"""
+    with create_local_queue(
+        client=admin_client,
+        name=request.param["name"],
+        cluster_queue=request.param["cluster_queue"],
+        namespace=kueue_enabled_notebook_namespace.name,
+        teardown=True,
+    ) as local_queue:
+        yield local_queue
+
+
+@pytest.fixture(scope="function")
+def kueue_notebook_persistent_volume_claim(
+    request: FixtureRequest,
+    kueue_enabled_notebook_namespace: Namespace,
+    unprivileged_client: DynamicClient,
+) -> Generator[PersistentVolumeClaim, Any, Any]:
+    """Create a PersistentVolumeClaim in the Kueue-enabled namespace"""
+    with PersistentVolumeClaim(
+        client=unprivileged_client,
+        name=request.param["name"],
+        namespace=kueue_enabled_notebook_namespace.name,
+        label={Labels.OpenDataHub.DASHBOARD: "true"},
+        accessmodes=PersistentVolumeClaim.AccessMode.RWO,
+        size="10Gi",
+        volume_mode=PersistentVolumeClaim.VolumeMode.FILE,
+    ) as pvc:
+        yield pvc
+
+
+@pytest.fixture(scope="class")
+def kueue_enabled_notebook_namespace(
+    request: FixtureRequest,
+    admin_client: DynamicClient,
+    unprivileged_client: DynamicClient,
+) -> Generator[Namespace, Any, Any]:
+    """Create a namespace with Kueue label enabled for notebook workloads"""
+    from utilities.infra import create_ns
+
+    namespace_name = request.param["name"]
+    add_kueue_label = request.param.get("add-kueue-label", True)
+
+    with create_ns(
+        admin_client=admin_client,
+        name=namespace_name,
+        unprivileged_client=unprivileged_client,
+        teardown=True,
+        add_kueue_label=add_kueue_label,
+        pytest_request=request,
+    ) as ns:
+        yield ns
+
+
+@pytest.fixture(scope="class")
+def patched_kueue_manager_config(
+    admin_client: DynamicClient,
+    request: FixtureRequest,
+) -> Generator[ConfigMap | None, Any, Any]:
+    """Conditionally patch the kueue-manager-config ConfigMap based on Kueue installation scenario
+
+    This fixture:
+    1. Detects the Kueue installation scenario
+    2. For RHOAI managed scenario: patches ConfigMap and restarts deployment
+    3. For Red Hat Kueue scenario: skips patching (no config/deployment present)
+    4. Yields the ConfigMap (or None if skipped) for tests
+    5. On teardown: restores original config if it was patched
+    """
+    # Detect Kueue installation scenario
+    scenario = detect_kueue_installation_scenario(client=admin_client)
+    LOGGER.info(f"Detected Kueue installation scenario: {scenario}")
+
+    # Check if we should patch config for this scenario
+    if not should_patch_kueue_config(scenario):
+        LOGGER.info(f"Skipping kueue-manager-config patching for scenario: {scenario}")
+        yield None
+        return
+
+    namespace = py_config["applications_namespace"]
```

Comment:
Please use else.

Reply:
Why? Isn't this way better readable? Am I missing something?